tests/plugins/release-monitor/index.js

```diff
@@ -0,0 +1,726 @@
+
+                nock("https://api.github.com")
+                    .get("/repos/test/repo-test/issues?labels=release%2Cpost-release")
+                    .reply(200, [{}]);
+
+                const newPrStatus = nock("https://api.github.com")
+                    .post("/repos/test/repo-test/statuses/111")
+                    .reply(200, assertPendingStatus);
+
+                await bot.receive({
+                    event: "pull_request",
+                    payload: {
+                        action: "opened",
+                        installation: {
+                            id: 1
+                        },
+                        pull_request: {
+                            number: 1,
+                            title: "New: add 1"
+                        },
+                        repository: {
+                            name: "repo-test",
+                            owner: {
+                                login: "test"
+                            }
+                        }
+                    }
+                });
+
+                expect(newPrStatus.isDone()).toBeTruthy();
+            });
+
+            test("put success for non semver patch PR under post release phase", async() => {
+                mockAllOpenPrWithCommits([
+                    {
+                        number: 1,
+                        title: "Fix: add 1",
+                        commits: [
+                            {
+                                sha: "111",
+                                commit: {
+                                    message: "Fix: add 1"
+                                }
+                            }
+                        ]
+                    }
+                ]);
+
+                nock("https://api.github.com")
+                    .get("/repos/test/repo-test/issues?labels=release%2Cpost-release")
+                    .reply(200, [{}]);
+
+                const newPrStatus = nock("https://api.github.com")
+                    .post("/repos/test/repo-test/statuses/111")
+                    .reply(200, assertSuccessStatusWithPendingRelease);
+
+                await bot.receive({
+                    event: "pull_request",
+                    payload: {
+                        action: "opened",
+                        installation: {
+                            id: 1
+                        },
+                        pull_request: {
+                            number: 1,
+                            title: "New: add 1"
+                        },
+                        repository: {
+                            name: "repo-test",
+                            owner: {
+                                login: "test"
+                            }
+                        }
+                    }
+                });
+
+                expect(newPrStatus.isDone()).toBeTruthy();
+            });
+
+            test("put success for all PR under non post release phase", async() => {
```

Comment:
Not fully sure I understand this test. Is this test for outside of a release cycle (i.e., no open "release" issue with or without "post-release")? And might be better to specify that the commit in this test is non-semver-patch.